lib/util.js

```diff
@@ -1481,3 +1481,24 @@ exports.getScrollBarWidth = function () {
 
   return (w1 - w2);
 };
+
+exports.topMost = function (pile, accessors) {
+  let candidate;
+  if (!Array.isArray(accessors)) {
+    accessors = [accessors];
+  }
+  for (const member of pile) {
+    candidate = member[accessors[0]];
```

Comment:
@eymiha It looks like this PR broke the develop branch.

`member` can be undefined. We need a check here: 
```js
if (!member) continue;
```